packages/core/scripts/severless-cloud-run/CloudRunHub.js

```diff
@@ -98,12 +99,12 @@ hub.post("/", async (req, res) => {
     let errorOutputs = [];
     let validOutputs = [];
     results.forEach((result, index) => {
-      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.value?.execResponse?.stderr) {
+      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code == "500") {
```

Comment:
Can error code be 4xx?

Reply:
it shouldn't be. the spoke only returns 200 or 500 errors. 

However, I agree though and I've changed it to result?.reason?.code != "200"